### ISSUE TITLE
[FEATURE] Applying model to a masked image

### ISSUE THREAD

**Issue by MosGeo** [MouseLand/cellpose#695]
**Is your feature request related to a problem? Please describe.**
I would like to apply the model on a masked image (ignoring some pixels in the input image). Is that possible in the current package? Any guidance?

**Describe the solution you'd like**
I am thinking of something like

```python
masks, flows, styles = model.eval(image, diameter=None, channels=[0, 0], mask=mask)
```

**Describe alternatives you've considered**
As I am new to this package, I still yet to learn about the details. I am thinking that it is possible to compute the neural network results, apply the masks and then continue with the segmentation but I am not sure if this is the best approach.
